src/main/kotlin/app/revanced/patches/finanzonline/detection/bootloader/patch/BootloaderDetectionPatch.kt

```diff
@@ -0,0 +1,37 @@
+package app.revanced.patches.finanzonline.detection.bootloader.patch
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Package
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patches.finanzonline.detection.bootloader.fingerprints.BootloaderDetection2Fingerprint
+import app.revanced.patches.finanzonline.detection.bootloader.fingerprints.BootloaderDetectionFingerprint
+
+
+@Patch
+@Name("remove-unlocked-bootloader-detection")
```

Comment:
Kind of verbose name. Could be simply shortened to `remove-bootloader-detection` or `bootloader-detection`.